src/main/kotlin/org/rust/cargo/toolchain/wsl/Utils.kt

```diff
@@ -14,6 +18,26 @@ fun WSLDistribution.expandUserHome(path: String): String {
     return "$userHome${path.substring(1)}"
 }
 
-fun Path.hasExecutableOnWsl(toolName: String): Boolean = pathToExecutableOnWsl(toolName).toFile().isFile
+fun Path.hasExecutableOnWsl(toolName: String): Boolean {
+    checkEdtAndReadAction()
+    return pathToExecutableOnWsl(toolName).toFile().isFile
+}
 
 fun Path.pathToExecutableOnWsl(toolName: String): Path = resolve(toolName)
+
+private val LOG: Logger = Logger.getInstance("org.rust.cargo.toolchain.Utils")
```

Comment:
```suggestion
private val LOG: Logger = Logger.getInstance("#org.rust.cargo.toolchain.Utils")
```

To be consistent with other logger objects